src/View.php

```diff
@@ -440,10 +450,10 @@ public function getHTML()
      * @link http://agile-ui.readthedocs.io/en/latest/js.html
      *
      * @param string|bool|null          $when     Event when chain will be executed
-     * @param array|jQuery_Chain|string $code     JavaScript chain(s) or code
+     * @param array|jQuery|string $code     JavaScript chain(s) or code
```

Comment:
Dark, where did i say that "when" could be an array? Give me link an i'll fix it there.